audio-worklet/test/bitcrusher-test.html

```diff
@@ -14,7 +14,7 @@
 -->
 <!DOCTYPE html>
```

Comment:
If you want to keep this file, please add a comment that says this will work when AudioWorkletNode works on OfflineAudioContext.